src/jsutils.h

```diff
@@ -57,21 +60,30 @@ size_t flash_strlen(const char *str);
 /// Copy a string from flash to RAM
 char *flash_strncpy(char *dest, const char *source, size_t cap);
 
+/// Compare a string in memory with a string in flash
+int flash_strcmp(const char *mem, const char *flash);
```

Comment:
Well, safe from having bad things happen :) I know what you mean.

Yeah, I guess I was thinking it'd simplify the binary search code (and also I bet there are a whole bunch of `strcmps` against constant strings that could easily be switched over for ESP8266).